helmfile/values/networkpolicy-common.yaml.gotmpl

```
@@ -51,8 +51,16 @@ certManager:
     {{- end }}
   letsencrypt:
     ips: {{- toYaml .Values.networkPolicies.certManager.letsencrypt.ips | nindent 6 }}
+
 falco:
   enabled: {{ and .Values.falco.enabled .Values.networkPolicies.falco.enabled }}
 
 rookCeph:
   enabled: {{ .Values.networkPolicies.rookCeph.enabled }}
+coredns:
+  enabled: {{ .Values.networkPolicies.coredns.enabled }}
+  externalDns:
+    ips: {{- toYaml .Values.networkPolicies.coredns.externalDns.ips | nindent 6 }}
```

I think you need to add this to make the config work
```suggestion
    ips: {{- toYaml .Values.networkPolicies.coredns.externalDns.ips | nindent 6 }}
  serviceIp:
    ips: {{- toYaml .Values.networkPolicies.coredns.serviceIp.ips | nindent 6 }}
```